tests/strategies/test_deepspeed_strategy.py

```diff
@@ -1269,13 +1269,19 @@ def test_deepspeed_with_meta_device(tmpdir):
 def test_deepspeed_multi_save_same_filepath(tmpdir):
```

Comment:
update this test instead of adding another one since deepspeed tests are expensive on ci.